docs/contribute/03_style.qmd

```diff
@@ -35,3 +35,59 @@ If you use `nix-shell`, all of these are already setup for you and ready to use,
 ## Docstrings
 We use [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) as our
 standard format for docstrings.
+
+## Documentation, blog, and other prose style
+
+Always capitalize Ibis in prose.
+
+### General points on style and word usage
+
+Avoid the passive voice.  Either Ibis does something or the user does
+  something or a particular backend does something.
+
+Try to avoid using words like "simply", "simple", "obviously", "just", when
+  describing things that Ibis makes easier.  Either the simplicity is
+  self-evident, or it isn't and you are inadvertently insulting the reader.
+
+LLMs do not write well, they write _correctly_.  These are not the same thing.
```

Comment:
this seems out of place, perhaps prose about "docs should be authored by you. we discourage LLM-written docs, but you're free to use tools to aid in your writing. be proud of what you submit idk"

Reply:
Yeah, I agree.